src/learnerPortalLinks.js

```diff
@@ -32,15 +32,16 @@ async function fetchLearnerPortalLinks(apiClient, userId) {
       const brandingConfiguration = enterpriseCustomer.branding_configuration;
       if (enableLearnerPortal && enterpriseCustomerSlug) {
         learnerPortalLinks.push({
-          title: `${enterpriseCustomer.name}`,
-          url: `${window.location.protocol}//${learnerPortalHostname}/${enterpriseCustomerSlug}`,
           // branding_configuration is not always returned as part
           // of the response so check if it exists before referencing fields
           branding_configuration: brandingConfiguration ? {
             logo: brandingConfiguration.logo || null,
             banner_border_color: brandingConfiguration.banner_border_color || null,
             banner_background_color: brandingConfiguration.banner_background_color || null,
           } : null,
+          title: `${enterpriseCustomer.name}`,
```

Comment:
nit: idk why we had `enterpriseCustomer.name` in the backticks here haha. Can probably remove those.